### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by awharn**
**What It Does**

Update dependencies to resolve technical currency

**How to Test**

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [ ] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Review by traeok — APPROVED** [review]
LGTM, thanks @awharn 😅

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,13 @@
 
 All notable changes to the IBM® Db2® Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Update follow-redirects transitive dependency to resolve technical debt [#146](https://github.com/zowe/zowe-cli-db2-plugin/pull/146)
```
Comment (anaxceron) [inline CHANGELOG.md:7]: Should be written in past tense. Missing a period at the end of the sentence.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,13 @@
...
+
 ## `4.1.12`
 
-- BugFix: Update follow-redirects transitive dependency to resolve technical debt
+- BugFix: Update follow-redirects transitive dependency to resolve technical debt [#138](https://github.com/zowe/zowe-cli-db2-plugin/pull/138)
```
Comment (anaxceron) [inline CHANGELOG.md:11]: Same as my first comment.

**Review by anaxceron — APPROVED** [review]
Two comments on minor edits.

**Review by gejohnston — APPROVED** [review]

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**